packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md

```diff
@@ -0,0 +1,345 @@
+1. The Okta authorization server validates the user’s access token and grants a new access token to the API1 service with scopes that allow that service to make requests to the API2 service. The new access token retains the user context so API2 service knows on whose behalf the request is being made.
+1. The API1 service makes a request to the API2 service with the new access token.
+
+> **Note:** You can determine which API service made API calls on behalf of which user by analyzing the Okta System Log or the looking for token grant events.
+
+## Set up token exchange
+
+The following sections walk you through the set up for an example token exchange flow using a single custom authorization server. For an example token exchange flow using more than one custom authorization server within an Okta tenant, see the [Trusted servers](#trusted-servers) section.
+
+> **Note:** You can use a service, web, native, and SPA app to exchange a token with a service app.
+
+### Create a native app integration
+
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. Select **OIDC - OpenID Connect**, and then **Native Application**.
+1. Name your application and at the bottom of the page select **Allow everyone in your organization to access**.
+1. Click **Save** and at the top of the page click **Back to Applications**.
+
+> **Note:** We recommend that native applications use the Authorization Code with PKCE authentication flow. See [Implement authorization by grant type](/docs/guides/implement-grant-type/authcodepkce/main/#create-the-proof-key-for-code-exchange) for more information on creating the Proof Key for Code Exchange (PKCE) for your native application.
+
+### Create a service app
+
+In token exchange use cases, an API microservice can act both as a resource server and a client. For this example, the native app gets an access token to make API requests to API1. API1 acts as the resource server. When API1 needs to talk to API2 on behalf of the user, the API1 becomes the OAuth client.
+
+Create a service app that represents API1.
+
+1. On the **Applications** page, click **Create App Integration**.
+1. Select **API Services**, and then click **Next**.
+1. Name your application. For this example, enter **API1**, and then click **Save**.
+1. In the **General Settings** section of the **General** tab, click **Edit** and select **Token Exchange** as the grant type for your application.
+1. Click **Save**.
+1. Copy the client ID in the **Client Credentials** section, and then copy the client secret in the **CLIENT SECRETS** section.
+1. [Base64 encode](/docs/guides/implement-grant-type/clientcreds/main/#flow-specifics) the client ID and client secret for use in the token exchange request from API1 to API2.
```

Comment:
The link maybe change to: 
-> `https://63ddd67b48ae0742e99ce475--reverent-murdock-829d24.netlify.app/docs/guides/implement-grant-type/clientcreds/main/#base64-encode-the-client-id-and-client-secret`

Reply:
I mean this part: `clientcreds/main/#base64-encode-the-client-id-and-client-secret`